portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,52 @@
+.menu ul {
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    list-style-type: none;
+}
+
+.menu ul li {
+    float: left;
+    padding: 0.5%;
+}
+
+.menu ul li:hover {
+    font-weight: bold;
+    border-bottom: 2px solid black;
+}
+
+.menu ul li a{
+    text-decoration: none;
+    text-align: center;
+    color: black;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 650px;
 }
 
 #greeting-container {
-  margin-top: 20px;
+    margin-top: 20px;
+}
+#home{
+    background: red;
+    height: 100vh;
+    width: 100vw;
```

Comment:
My version of Chrome adds an 8pm margin for the `body` element when I view your page, so a width of `100vw` means that get the tiniest horizontal scroll bars. I'd experiment with different width units if you're experiencing the same thing. (I think your intent is to take up the full width of the user's browser, but no more.)